NEWS.md

```diff
@@ -5,3 +5,47 @@
 * Prevent `init()` in home directory.
 * Included `build_js()` and `build_sass()` in template CI.
 * Use R version from the lockfile in CI.
+
+# rhino 1.1.1
+
+## What's Changed
+* Add upcoming events section to the README by @kamilzyla in https://github.com/Appsilon/rhino/pull/311
+* Update upcoming events by @kamilzyla in https://github.com/Appsilon/rhino/pull/315
+* Update info on upcoming events by @kamilzyla in https://github.com/Appsilon/rhino/pull/316
+* Upgrade roxygen2 to 7.2.1 and regenerate documentation by @kamilzyla in https://github.com/Appsilon/rhino/pull/320
```

Comment:
Kudos for adding older releases! However the auto-generated lists of PRs are too detailed and not very meaningful for our users IMO. I'll create a separate task to write a nicer changelog for older versions. Let's keep it minimal for now.
```suggestion
Minor release to fix CRAN check failures (upgrade roxygen2 to 7.2.1 and regenerate documentation).
```

Reply:
Removed What's Changed.